test/services/PostJsonModelToPatchSpec.scala

```diff
@@ -0,0 +1,38 @@
+package services
+
+import org.scalatest.{ FreeSpec, Matchers, PrivateMethodTester }
+import play.api.libs.json.JsString
+import uk.gov.ons.sbr.models._
+import uk.gov.ons.sbr.models.edit.{ Operation, OperationTypes }
+
+class PostJsonModelToPatchSpec extends FreeSpec with Matchers with PrivateMethodTester {
```

Comment:
Why do we need to test private methods?

Personally, this is not something that I have *ever* done.
Either the private method is an internal implementation detail, and we can simply concentrate on testing the observable inputs / outputs of the higher-level routine.  Or the private method is actually really important behaviour - in which case it should probably be extracted out, tested independently, and then injected into anything that needs it.

This is what TDDers refer to as "listening to the tests".  The argument is, if you feel you need to do this, the design can probably be improved.  This highlights that TDD is not about the T (testing) - it is about using tests to DD (drive the design).

Reply:
That all makes sense, I guess I wanted to be thorough when it came to important methods such as forming the path. I'll remove it.